README.md

```diff
@@ -39,31 +39,25 @@ EXAMPLES
 
 INSTALLATION AND CONFIGURATION
 ------------------------------
-
-
-1. Unzip the tarball to a directory of your choice, typically `/usr/local/`
-2. A new directory called easy-transform-metadata-<version> will be created
-3. Add the command script to your `PATH` environment variable by creating a symbolic link to it from a directory that is
-   on the path, e.g. 
-   
-        ln -s /usr/local/easy-transform-metadata-<version>/bin/easy-transform-metadata /usr/bin
-
-
-
-General configuration settings can be set in `cfg/application.properties` and logging can be configured
-in `cfg/logback.xml`. The available settings are explained in comments in aforementioned files.
-
+Currently this project is build only as an RPM package for RHEL7/CentOS7 and later. The RPM will install the binaries to
+`/opt/dans.knaw.nl/easy-transform-metadata `, the configuration files to `/etc/opt/dans.knaw.nl/easy-transform-metadata `,
+and will install the service script for `systemd`. 
 
 BUILDING FROM SOURCE
 --------------------
-
 Prerequisites:
 
 * Java 8 or higher
 * Maven 3.3.3 or higher
+* RPM
 
 Steps:
-
-        git clone https://github.com/DANS-KNAW/easy-transform-metadata.git
-        cd easy-transform-metadata
-        mvn install
+    
+    #!bash
```

Comment:
Why is this line necessary?

Reply:
Oops copied it from `easy-bag-store`, which has been converted to `mkdocs`, where this has meaning. I'll remove it.

Reply:
no you didn't remove it; lol. You even added it to [`easy-module-archetype`](https://github.com/DANS-KNAW/easy-module-archetype/pull/47/files#diff-68e651711b2b31f340c37ee386107dd7R54)